fastlane/lib/fastlane/plugins/template/.rubocop.yml

```diff
@@ -0,0 +1 @@
+../../../../../.rubocop.yml
```

Comment:
I'd say we copy over the `.rubocop.yml`, because if fastlane changes the design guidelines, we don't want it to break plugins. I'll also try to reduce the amount of custom rules in there now 👍